lib/rights_api/services.rb

```diff
@@ -8,4 +8,12 @@ module RightsAPI
   Services.register(:rights_database) do
     RightsDatabase
   end
+
+  Services.register(:db_connection) do
+    connection = Services[:rights_database].connect
+    unless ENV["RIGHTS_API_NO_LOG"]
```

Comment:
Consider having `Services.register(:logger)` and using that from `db_connection` (in case we want to use the same logger elsewhere)